packages/core/useTransition/index.ts

```diff
@@ -115,6 +122,68 @@ function createEasingFunction([p0, p1, p2, p3]: CubicBezierPoints): EasingFuncti
   return (x: number) => (p0 === p1 && p2 === p3) ? x : calcBezier(getTforX(x), p1, p3)
 }
 
+const lerp = (a: number, b: number, alpha: number) => a + alpha * (b - a)
+
+const toVec = (t: number | number[] | undefined) => (isNumber(t) ? [t] : t) || []
+
+/**
+ * Transition from one value to another.
+ *
+ * @param source
+ * @param from
+ * @param to
+ * @param options
+ */
+export function transition<T extends number | number[]>(
```

Comment:
The name to be is a bit too general to me, can we have a more explicit name, like

`createTransitionPromise` or `createTransition`?

Reply:
Personally, I like the name `transition` because it works as a compliment to `useTransition`. In other words, it's what `useTransition` is actually using. I'm open to changing though, but I don't think "create" is a good fit, since it operates on an existing ref rather than creating anything. Maybe something like `transitionRef`, `executeTransition`, `runTransition`, or `performTransition` could work...

I think of those choices, `transitionRef` is probably my favorite. I'm not sure though, what do you think? 🤔

Reply:
I am ok with `performTransition` or `executeTransition`.

But `transitionRef` can be confusing as we have `deboundedRef` etc, which returns a `ref` - but in our case we didn't right?

Reply:
I had time to try these names out over the weekend and I think `executeTransition` felt nicer, I'll make the change 🙂